README.md

```diff
@@ -1,55 +1,63 @@
+- [Admin Consent](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.html) must be granted in the DCP admin app.
 
-![Items](https://user-images.githubusercontent.com/109528797/186420636-61efa334-6041-44df-9056-11e6cf44da78.png)
 
-* Open the "memory files" tab and create memory files named Teams, Channels and Chats. The purpose of these files is to persist the created Teams, Channels and Chats.
+## Examples
 
-![Items](https://user-images.githubusercontent.com/109528797/186420786-f9b95bca-ee82-49d6-901a-5a19c4c14f43.png)
+### Microsoft Teams
 
-* Also create a memory file named YesOrNo with the following entries.
+#### Installation
 
-![Items](https://user-images.githubusercontent.com/109528797/186423143-bab59820-8e45-4315-a192-24b34398b502.png)
+- [Deploy the DMAPP]() to your DataMiner System. Installing this package, will add the automation scripts for Microsoft Teams in the Automation module.
 
-* Connect your dataminer system to the cloud if it is not yet [Cloud Connected](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Connecting_your_DataMiner_System_to_the_cloud.html).
+- [Access the DataMiner System](https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner.html) and open the Automation module (via apps > Automation). <details><summary>...</summary>
+![Gif-Automation](https://user-images.githubusercontent.com/109528797/186685478-9eac1cbf-f2d9-4c9a-8a6a-a2f499dbdcd9.gif)
 
-* Grant access with [Admin Consent](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.html).
+</details>
 
+- Open the "memory files" tab and create memory files named *Teams, Channels and Chats*. The purpose of these files is to persist the created Teams, Channels and Chats.<details><summary>...</summary>
+![Gif-MemFiles](https://user-images.githubusercontent.com/109528797/186685736-dacafe23-53be-4165-8982-eb2113549d78.gif)
 
+</details>
 
-### Usage
+- Also create a memory file named YesOrNo with the following entries: *{postion:0, value:1, description:yes ; postion:1, value:0, description:no}*<details><summary>...</summary>
+![Gif-MemFilesYesorNo](https://user-images.githubusercontent.com/109528797/186685771-9d9c4155-1f58-4700-98aa-90ebd19c329e.gif)
 
-* Open the Automation module (via apps > Automation) and click on the "automation scripts" tab and double click the "Create Team" automation script. Fill in the fields for creating a new Team, **the fields are case sensitive.**
-* When pressing the "execute now" button, a Team is created in Microsoft Teams with the Dataminer bot.
+</details>
 
-<img src="https://user-images.githubusercontent.com/109528797/186440501-4850222e-c503-40fc-8f9a-003e8c7d5a30.png" width="684" height="757">
 
+#### Usage
```

Comment:
Also for create channels, add member/owner and send channel notification ? or have only this one for create team and then do a more generic one like, 'repeat these steps for the other scripts'?

Maybe we have to explain you can send messages in a Channel or chat
A channel exists in a Team
A Chat is a group or private chat (no create support yet)